src/cli_plugin/install/index.js

```diff
@@ -47,6 +47,7 @@ export default function pluginInstall(program) {
     .command('install <plugin/url>')
     .option('-q, --quiet', 'disable all process messaging except errors')
     .option('-s, --silent', 'disable all process messaging')
+    .option('--no-optimize', 'disable optimization after plugin extraction')
```

Comment:
`--skip-optimize` was mentioned [here](https://github.com/elastic/kibana/pull/16302#discussion_r164524032), but I am OK with `--no-optimize` as the option.

Reply:
++ for Commander convention.